src/util/createStoreServer.js

```diff
@@ -30,7 +30,7 @@ export const serverFetchQueries = request => () => queries =>
  */
 export function getServerCreateStore(routes, middleware, request, baseUrl) {
 	const middlewareToApply = [
-		catchMiddleware,
+		catchMiddleware(err => request.server.app.logger.error(err)),
```

Comment:
just an update from the current `if (typeof window === 'undefined')` structure in order to use the new and improved server-side logger on the server - not related to routing update, so I can break it out into a separate PR if it's preferable.